client/components/seo-preview-pane/index.jsx

```diff
@@ -181,10 +181,10 @@ export class SeoPreviewPane extends PureComponent {
 						</h1>
 						<p className="seo-preview-pane__description">
 							{ translate(
-								`Below you'll find previews that ` +
-								`represent how your post will look ` +
-								`when it's found or shared across a ` +
-								`variety of networks.`
+								'Below you\'ll find previews that ' +
```

Comment:
In case of single quotes we [can use double quotes](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#strings), escaping them is not pretty.

In this case it might be better to use the proper typographical apostrophe, though: you’ll.